config/locales/en.yml

```diff
@@ -924,6 +925,7 @@ en:
         legend: What is your client's employment status?
         hint: "You cannot use this service if your client, or their partner, is self-employed."
       passporting:
+        definition: Help with passporting benefits
         legend: Does your client receive a passporting benefit?
         partner_hint: Also select yes if your client is named on their partner’s passporting benefit claim.
```

Comment:
@starswan a question for UCD, but do we still need this text as it wasn't included in the new design? 

> Also select yes if your client is named on their partner’s passporting benefit claim.

Reply:
@starswan had confirmation from UCD that this line now goes 👍🏽

Reply:
done